npm start & npm run console scrits are broken on version 0.13.7

With the new version update it is impossible to start the application. 
I realize this matter when I runned a  `npm install` on my **Patternplate** project, at that moment the application was impossible to launch.
When you try to run `npm start` the server throws the following `Error: Cannot find module 'denodeify't`.
The same error is thrown  when you try to build with `npm run console`.

This has major repercussions for all of us using  **Patternplate**.  The quick fix for us was to manually move one version down **0.13.6** on the node_modules directory where the **Patternplate** project resides.

Appreciate your comments.

--- a/package.json
+++ b/package.json
@@ -62,6 +62,7 @@
     "btoa": "^1.1.2",
     "checksum": "^0.1.1",
     "core-util-is": "^1.0.1",
+    "denodeify": "^1.2.1",
     "envify": "^3.4.0",
     "find-root": "^0.1.1",
     "git-rev": "^0.2.1",

